README.md

```diff
@@ -548,6 +548,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Bhavanshu Jain](https://github.com/bhavanshu-1112)
   - [Bhavsagar Chaudhary](https://github.com/itsbhavsagar)
   - [Bhavya Giri](https://github.com/bhavya-giri)
+  - [Bhavneet singh](https://github.com/Bhavneet-singh)
```

Comment:
Please add your name in line 549, after Bhavanshu Jain.
Let us know if you need any help with that. 🙂